Code/MinimalLib/minilib.cpp

```diff
@@ -954,3 +954,51 @@ JSLog *set_log_capture(const std::string &log_name) {
 void enable_logging() { RDKit::MinimalLib::LogHandle::enableLogging(); }
 
 void disable_logging() { RDKit::MinimalLib::LogHandle::disableLogging(); }
+
+#ifdef RDK_BUILD_MINIMAL_LIB_RGROUPDECOMP
+int JSRgroupDecomp::add(const JSMol &mol) {
+  return decomp.add(RDKit::RWMol(*mol.d_mol));
+}
+
+bool JSRgroupDecomp::process() {
+  return decomp.process();
+}
+
+std::pair<std::vector<std::string>, std::vector<JSMolList*>> JSRgroupDecomp::getRGroupsAsColumns() const {
+  RGroupColumns cols = decomp.getRGroupsAsColumns();
+  size_t length = cols.size();
+  std::vector<std::string> keys(length);
+    std::vector<JSMolList*> res(length);
+  size_t i = 0;
+  for (std::map<std::string, RGroupColumn>::iterator it = cols.begin(); it != cols.end(); ++it) {
+    keys[i] = (it->first);
+    res[i] = new JSMolList(cols[keys[i]]);
```

Comment:
can we use it->second instead of cols[keys[i]]? Just a small idea and anyway not that important.